### ISSUE TITLE
GitLens intentionally hijacking Source Control panel name / icon

### ISSUE THREAD

**Issue by lsegal** [gitkraken/vscode-gitlens#1880]
It seems like GitLens is hijacking the "Source Control" panel name and icon in what seems like an attempt to intentionally confuse users into thinking it is the default source control panel that comes with VS Code.

I'm sure GitLens' source controls provides useful functionality, but I am a little suspect about this behavior, especially since the latest update seems to also inject the panel into the sidebar as a default panel right next to VS Code's Source Control, and, for the nail in the coffin, even uses the exact same icon. I cannot imagine any other way to interpret this than an intentional obfuscation of these two panels. I'll admit I was initially very confused by this and thought that "Worktrees" were a VS Code feature; I can only imagine less experienced users would probably be even more so.

![image](https://user-images.githubusercontent.com/686/156482126-4e16f49b-bb7d-49a5-a8c3-00e4b7058b1e.png)

In fact, because of the way VS Code operates, even the side bar context menu is confused, assigning the "Ctrl+Shift+G" hotkey to the "Source Control" panel that does not belong to VS Code, but rather, GitLens. When you attempt to hide the panel that does not have the hotkey, you end up hiding VS Code's. This is how you have to configure the sidebar:

![image](https://user-images.githubusercontent.com/686/156483147-9ec17e89-692c-43a1-b73b-fe866e6d8b69.png)

For reference, I verified that Ctrl+Shift+G is indeed configured to show VS Code's source control panel, not GitLens. Hitting Ctrl+Shift+G does in fact focus on VS Code's panel.

![image](https://user-images.githubusercontent.com/686/156483256-6bad4ab3-9ff7-42f2-8ff7-dadecf8054a2.png)

tldr; If GitLens is going to have its own "Source Control" panel, it should be appropriately named (prefixed with "GitLens") and its own icon so as to not cause confusion.

- GitLens Version: 12
- VSCode Version: 1.64.2
- OS Version: Windows 11

Steps to Reproduce:

1. Install GitLens 12.x

**Comment by rotarydialer** [comment]
Confirming this behavior on Mac. Very confusing, and concerning enough to lead me to disable GitLens. Likely to remove unless this turns out to have been an innocent mistake.

**Comment by aexvir** [comment]
yeah, so disappointed to see these abusive practices starting with the new major version
uninstalled and not really looking forward to get it back 😕

**Comment by eamodio** [comment]
There are a few things going on here, so let me break things up.

First, let's talk about the overriding of the Source Control keyboard shortcut. To be clear this is NOT new, and certainly not trying to be deceptive in any way -- it's been this way in GitLens for many years when you use the default "chorded" keyboard shortcuts. GitLens remaps `ctrl+shift+g` to be the starting chord for many commands -- and makes `ctrl+shift+g, g` the chord to open the Source Control view. Originally this chord was `ctrl+alt+g` but that caused all sorts of issues on Windows machines, so I switched it to `ctrl+shift+g`. I certainly won't defend that overriding that shortcut key was a good decision, but after the amount of pain it seemed to cause when I switched away from `ctrl+alt+g`, I've always been very hesitant to change it (I don't want to break people who rely on it and if you are comfortable with chording, then the extra `g` to get to the SCM view isn't much of an issue). After that, I also added a setting for users to switch between different "keyboard" modes for GitLens -- the default is "chorded", but there is also an `alt`-based scheme, and an option to have GitLens set no shortcuts at all. In neither the `alt` or no shortcuts options are any other shortcuts overridden. Again, I'm not trying to defend overriding that shortcut key (as if I knew today what I knew then, I would avoid it), but that there is a lot of legacy and muscle memory I don't want to break.

As for the duplicate Source Control view, that is not something GitLens does at all -- GitLens by default adds a "GitLens" side bar for its "Home" view, and adds all of the rest of its views into the Source Control side bar. GitLens also provides a command for users to change that behavior and add all of the GitLens views into the GitLens side bar. What seems to be happening here, is that you've dragged some view around and VS Code created another "Source Control" view, as VS Code has a bunch of heuristic for what happens when you drag a view out of one side bar into a new one. And to be honest, it does some weird things in certain cases -- and there are also some known bugs in this area too (one I've been trying to get VS Code to fix for a while, where the wrong GitLens icon is used sometimes depending on which views you move around).

To resolve this issue, you can right-click on each view and select "Reset Location" (if that menu choice doesn't show up, it's because the view is in its "default" location). Or you can run the "Reset View Locations" command from the Command Palette to reset ALL views to their default location.

Hope that helps.
